### PR TITLE
Add astar config

### PR DESCRIPTION
**Pull request by Lohann**
## Description

Adds Astar Config to Ethereum connector

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Code review prechecks:

- [x] Code follows the style guidelines of this project
- [x] Code has been self-reviewed
- [x] Inline comments have been added for each method
- [x] I have made corresponding changes to the documentation
- [x] Code changes introduces no new problems or warnings
- [x] Test cases have been added
- [x] Dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by dvc94ch — APPROVED** [review]
LGTM. We should also add a config for the Ethereum mainnet

File: chains/astar/Dockerfile [hunk 1]
```diff
@@ -3,6 +3,9 @@ ARG IMAGE_VERSION
 ARG VCS_REF
 ARG BUILD_DATE
 
+FROM alpine:latest AS certs
+RUN apk --update add ca-certificates
```
Comment (Lohann) [inline chains/astar/Dockerfile:7]: I'll remove this once we make sure that all creates are using `webpki_rustls`

File: chains/ethereum/config/src/lib.rs [hunk 2]
```diff
@@ -7,35 +7,98 @@ use std::sync::Arc;
...
-        connector_port: 8081,
-        testnet: network == "dev",
-    })
+    let config = match network {
+        "dev" => BlockchainConfig {
+            blockchain: "ethereum",
+            network: "dev",
+            algorithm: Algorithm::EcdsaRecoverableSecp256k1,
+            address_format: AddressFormat::Eip55,
+            coin: 1,
```
Comment (dvc94ch) [inline chains/ethereum/config/src/lib.rs:16]:
```suggestion
            coin: if network == "mainnet" { 60 } else { 1 },
```

**Review by dvc94ch — APPROVED** [review]

File: chains/ethereum/config/src/lib.rs [hunk 3]
```diff
@@ -8,62 +8,42 @@ use std::sync::Arc;
 pub fn config(network: &str) -> Result<BlockchainConfig> {
     anyhow::ensure!(network == "dev");
     let config = match network {
-        "dev" => BlockchainConfig {
+        "dev" | "mainnet" => BlockchainConfig {
             blockchain: "ethereum",
             network: "dev",
```
Comment (dvc94ch) [inline chains/ethereum/config/src/lib.rs]: if network == "dev" { "dev" } else { "mainnet" }

**Pull request merged**